includes/ajax.php

```diff
@@ -393,15 +393,24 @@ function tsml_ajax_geocodes()
     //get lookups, todo consider adding regions to this
     $locations = $groups = [];
     $all_locations = tsml_get_locations();
```

Comment:
I just cleaned up some of the formatting here to better read it. No code change.